src/app/modules/group/http-services/request-actions.service.ts

```diff
@@ -4,6 +4,7 @@ import { ActionResponse, successData, objectToMap } from 'src/app/shared/http-se
 import { map } from 'rxjs/operators';
 import { Observable } from 'rxjs';
 import { appConfig } from 'src/app/shared/helpers/config';
+import { Result } from '../helpers/response-toast';
```

Comment:
A service cannot depend on response toast things :-/ But you can return `{ countRequests: int, countSuccess: int }` without importing it.

Reply:
fixed